addons/mail/views/res_config_settings_views.xml

```diff
@@ -111,6 +111,22 @@
                                 </div>
                             </div>
                         </div>
+                        <div class="col-12 col-lg-6 o_setting_box">
+                            <div class="o_setting_right_pane">
+                                <span class="o_form_label">Link Preview Throttle</span>
+                                <div class="row">
+                                    <div class="text-muted col-md-12">
+                                        Configure link preview throttle (0 to disable link preview)
+                                    </div>
+                                </div>
+                                <div class="content-group">
+                                    <div class="row mt16" id="mail_link_preview_throttle">
+                                        <label string="Value" for="link_preview_throttle" class="col-lg-3"/>
+                                        <field name="link_preview_throttle" placeholder="99"/>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
```

Comment:
Probably not necessary in settings, however what would be great to have is the technical views for link preview in debug mode.